front/index.html

```diff
@@ -86,6 +87,75 @@
             vertical-align: top;
         }
 
+        /* Splash-screen */
+
+        #splash-screen {
+            display: block;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+
+            background-color: rgb(50, 50, 50);
+            opacity: 0.9;
+            z-index: 1;
```

Comment:
Good for now, but to keep in mind
To keep in mind https://twitter.com/ryanflorence/status/456512425102028800